charts/prometheus/README.md

```diff
@@ -60,11 +60,24 @@ A [`values.schema.json`](https://helm.sh/docs/topics/charts/#schema-files) file
 ## Upgrading Chart
 
 ```console
-helm upgrade [RELEASE_NAME] [CHART] --install
+helm upgrade [RELEASE_NAME] prometheus-community/prometheus --install
 ```
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 23.0
+
+Releases of the included subcharts (dependencies) have been bumped as follows:
+
+| Subchart                    | Release | Previous Release |
+|--------------------------|---------|------------------|
+| [alertmanager](https://github.com/prometheus-community/helm-charts/releases/tag/alertmanager-0.31.0)       | 0.31.0   | 0.30.1 |
+| [kube-state-metrics](https://github.com/prometheus-community/helm-charts/releases/tag/kube-state-metrics-5.6.3)       | 5.6.3   | 4.30.0 |
+| [prometheus-node-exporter](https://github.com/prometheus-community/helm-charts/releases/tag/prometheus-node-exporter-4.17.3) | 4.17.3 | 4.8.1 |
+| [prometheus-pushgateway](https://github.com/prometheus-community/helm-charts/releases/tag/prometheus-pushgateway-2.1.6)   | 2.1.6 | 2.0.4 |
+
+Users are advised to review changes in the corresponding chart releases before upgrading.
+
```

Comment:
@zeritti 
do we need to provide some way to tell how to export the user current data and import to the new version?

Reply:
@monotek 
templates/pvc.yaml, in case the user enable PVC.

Reply:
@zanhsieh There is no issue related to the pushgateway's data stored in a persistent volume during the upgrade, the volume/its contents are not affected and no change on the pushgateway's side in this respect. 

However, whilst upgrading the pushgateway subchart with _deployment_ and persistent volume enabled is straightforward, not so with _statefulset_ and persistent volume - the latter cannot be patched due to changing labels and the upgrade fails. Fo this scenario, I'll have to add a note on upgrading - deleting the pushgateway's statefulset is needed before upgrading.

We should consider whether including the full label set from `prometheus-pushgateway.defaultLabels` (with `app.kubernetes.io/version`) in statefulset's `spec.volmeClaimTemplates` is reasonable as it leads to requiring manual intervention at an upgrade.

Reply:
@zeritti 
Thank you for testing that out. I think put some warning text as for pushgateway subchart sts with pvc as you suggested should be fine. We only do due diligence since in that case the user decided to go with sts - that's their choice.